### PR TITLE
feat: add polytone

### PR DESCRIPTION
**Pull request by Reecepbcups**
ref: #149

The initial migration of Polytone over with reviewers added from the DAODAO team. Releases CI will be done in another PR since it does not work in main currently (we can either do on every push to main, just optimize, or just on tagged releases (prefered))

### CODE REVIEW

**Comment by Reecepbcups** [comment]
Blocked on coordination w/ getting them in as codeowners, copying over the wiki, and handling releases

File: cosmwasm/polytone/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+rpath = false
+
+[workspace.dependencies]
+polytone = { path = "packages/polytone", version = "1.0.0" }
+polytone-proxy = { path = "contracts/main/proxy", version = "1.0.0" }
+polytone-note = { path = "contracts/main/note", version = "1.0.0" }
+polytone-voice = { path = "contracts/main/voice", version = "1.0.0" }
+
+cosmwasm-schema = "1.2.1"
+cosmwasm-std = { version = "1.5.0", features = ["ibc3"] }
```
Comment (JakeHartnell) [inline cosmwasm/polytone/Cargo.toml:29]: Make sure you have the latest upstream, we bumped some dependencies: https://github.com/DA0-DA0/polytone/pull/63#pullrequestreview-2077612554

Comment (Reecepbcups) [reply cosmwasm/polytone/Cargo.toml:29]: also ref: https://github.com/DA0-DA0/polytone/pull/65

Comment (Reecepbcups) [reply cosmwasm/polytone/Cargo.toml]: Pulled in all at b459f279c07b48316a75b2b5d6bbdb7030b3a6d0. Excludes the releases & audit dep workflows atm. Will test those outside this repo to ensure only tags for polytone get released for.

**Comment by Reecepbcups** [comment]
cc @NoahSaso / @JakeHartnell All workflows are in, latest v1.1.0 as well. 

Releases tested in my personal repo (https://github.com/Reecepbcups/ibc-apps/releases). _The 1.1.3 tag is just unique so I did not have to delete tags while testing workflows_
image

All that is left is pulling over issues & the wiki copy paste. I'll tag v1.1.0 here once merged to main :)

**Comment by Reecepbcups** [comment]
The remaining audit dep check is an upstream wasm import 

```
    ... which satisfies dependency `ed25519-zebra = "^3"` (locked to 3.1.0) of package `cosmwasm-crypto v1.5.5`
    ... which satisfies dependency `cosmwasm-crypto = "^1.5.5"` (locked to 1.5.5) of package `cosmwasm-std v1.5.5`
```

This repo uses the 4.1.3 safely. Safe to merge

**Pull request merged**